test/controllers/api/stages_controller_test.rb

```diff
@@ -0,0 +1,99 @@
+require_relative '../../test_helper'
+
+SingleCov.covered!
+
+describe Api::StagesController do
+  def assert_changed(model, method, amount)
+    initial = model.send(method).to_i
+    yield
+    initial.wont_equal model.send(method), "No change detected."
+    initial.must_equal (model.send(method) - amount), "#{model} was expected to change by #{amount}"
+  end
+
+  def assert_not_changed(model, method, amount = 0)
```

Comment:
yep

On Tue, Aug 9, 2016 at 11:25 AM, Adam Panzer <notifications@github.com>
wrote:

> In test/controllers/api/stages_controller_test.rb
> <https://github.com/zendesk/samson/pull/1163#discussion_r74116435>:
>
> > @@ -0,0 +1,99 @@
> > +require_relative '../../test_helper'
> > +
> > +SingleCov.covered!
> > +
> > +describe Api::StagesController do
> > +  def assert_changed(model, method, amount)
> > +    initial = model.send(method).to_i
> > +    yield
> > +    initial.wont_equal model.send(method), "No change detected."
> > +    initial.must_equal (model.send(method) - amount), "#{model} was expected to change by #{amount}"
> > +  end
> > +
> > +  def assert_not_changed(model, method, amount = 0)
>
> Okay to use assert_difference("Stage.count", 0)?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1163/files/ec9176e57ff6e9773ea877bdef9761f323850b46#r74116435>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZ4EgILUDJEiTEG12_SyixAMLi_lXks5qeMYtgaJpZM4JVi_g>
> .
>